src/app/beer_garden/log.py

```diff
@@ -169,7 +171,7 @@ def load(cls, filename: str, default_config) -> None:
         Returns:
             None
         """
-        if filename:
+        if filename and os.path.exists(filename):
```

Comment:
This needs to be backed out. If an incorrect file is specified an error should be raised. This change will make the system silently fall back to the default configuration.

Reply:
What if instead we still raise the error but allow them to proceed with the defaults. That way if they for some reason go back later and add the configuration they don't need to restart Beer Garden to apply it.

Reply:
I backed it out and created a new PR for that change. #587